ui/pages/settings/flask/snaps-list-tab/view-snap/view-snap.container.js

```diff
@@ -0,0 +1,35 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { getSubjectMetadata } from '../../../../../selectors';
+import { removePermittedAccount } from '../../../../../store/actions';
+import ViewSnap from './view-snap.component';
+
+const mapStateToProps = (state, ownProps) => {
+  const { snap } = ownProps;
+  const subjectMetadata = getSubjectMetadata(state);
+  const connectedSubjects = [];
+  Object.entries(subjectMetadata).forEach(([_, entry]) => {
```

Comment:
This could probably be a `reduce` instead of `forEach + connectedSubjects` but no big deal, we can improve that in the future.